puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/foreman-proxy-develop-test.groovy

```diff
@@ -0,0 +1,89 @@
+def commit_hash = ''
+
+pipeline {
+    agent any
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+
+    stages {
```

Comment:
Where did Ruby 2.0 go? EL7 on production still runs Ruby 2.0 so it's still a very important target.